Resources/Locale/en-US/prayers/prayers.ftl

```diff
@@ -0,0 +1,7 @@
+﻿prayer-verbs-subtle-message = Subtle Message
+prayer-verbs-pray = Pray
+prayer-chat-notify = PRAYER: {$username}: {$message}
+
+prayer-popup-subtle-default = You hear a voice in your head...
+prayer-popup-notify-sent = Your message has been sent to the gods...
+prayer-popup-notify-locked = You are not worthy to pray to me.
```

Comment:
This seems like someone's actually talking to the person. Say something like "You don't feel worthy..." instead.

Reply:
the first line is intentional because that's the admin's subtle popup